pkgs/top-level/python-packages.nix

```diff
@@ -5391,6 +5397,8 @@ in {
 
   flask_oauthlib = callPackage ../development/python-modules/flask-oauthlib { };
 
+  flask_paginate = callPackage ../development/python-modules/flask-paginate { };
```

Comment:
I would use hyphens, especially if the project itself also uses hyphens in the name.

Reply:
I think it's somewhat of a policy to prefer `_` over `-` in nixpkgs, because using whatever the package is using gets confusing fast ("Why does foo_bar not exist, oh right it's foo-bar", "Why does bar-foo not exist, oh right it's bar_foo").